### PR TITLE
feat: Add font awesome icon to project #P2022-610

### PR DESCRIPTION
**Pull request by rast-mk1**
To use Font Awesome icons:

first `import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';` in your page
and `import { faRuler } from '@fortawesome/free-solid-svg-icons';` specific icons

then `<FontAwesomeIcon icon={faRuler} />` use your icon

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import Layout from 'components/Layout';
 import '../styles/globals.css'
+import { config } from '@fortawesome/fontawesome-svg-core'
+import '@fortawesome/fontawesome-svg-core/styles.css'
+config.autoAddCss = false
```
Comment (danielmajewski) [inline pages/_app.js:5]: is it possible to make seperate file for this import and configuration? E.g. `fonts.js` and just `import '..../fonts', something like that?

Comment (rast-mk1) [reply pages/_app.js:5]: will look into this, thanks

File: pages/fontawesome.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { config } from '@fortawesome/fontawesome-svg-core';
+import '@fortawesome/fontawesome-svg-core/styles.css';
+config.autoAddCss = false;
```
Comment (DianaSzalek) [inline pages/fontawesome.js:3]: Please move all of this into `global.css` that shouldn't be a page

Comment (rast-mk1) [reply pages/fontawesome.js:3]: Yes, I am now aware of that. Will fix it. Thank you!

**Pull request closed without merging**